### ISSUE TITLE
macOS 10.12: Error installing rmlint with homebrew, during `scons` build (?)

### ISSUE THREAD

**Issue by alyssadai** [sahib/rmlint#580]
System details:
- macOS 10.12
- XCode 9.2

Summary:
`brew install rmlint` successfully installs dependencies, but appears to halt at the scons build stage:
```shell
==> Installing rmlint
==> scons config
==> scons
Last 15 lines from /Users/abcdresearch/Library/Logs/Homebrew/rmlint/02.scons:
lib/utilities.c:1152:50: error: use of undeclared identifier 'mount'; did you mean 'mounts'?
            best2 = g_unix_mount_get_device_path(mount);
                                                 ^~~~~
                                                 mounts
lib/utilities.c:1142:12: note: 'mounts' declared here
    GList *mounts = g_unix_mounts_get(NULL);
           ^
lib/utilities.c:1157:46: error: use of undeclared identifier 'g_unix_mount_free'
    g_list_free_full(mounts, (GDestroyNotify)g_unix_mount_free);
                                             ^
6 errors generated.
scons: *** [lib/utilities.o] Error 1
Cannot import `sphinx_bootstrap_theme`; falling back to `nature`.
^ This is no error, will cause only slightly different html output.
scons: building terminated because of errors.

Do not report this issue to Homebrew/brew or Homebrew/core!


Error: You are using macOS 10.12.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version.
```

Not sure if the older version of the OS is the culprit (I'm working with a shared lab machine), but any suggestions for troubleshooting this would be much appreciated!

**Comment by Cebtenzzre** [comment]
The full contents of ` ~/Library/Logs/Homebrew/rmlint/02.scons` would be helpful, but from the part you've shown it looks like scons is unable to find gio-unix-2.0 and `gio/gunixmounts.h` is not included. What is the output of these commands?
```
pkg-config --path glib-2.0
pkg-config --path gio-2.0
pkg-config --path gio-unix-2.0
```
